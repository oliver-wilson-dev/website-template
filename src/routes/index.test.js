jest.mock('../helpers/lazyLoadComponent', () => ({ importFn }) => {
  importFn();
  const LazyLoadedComponent = () => null;

  return LazyLoadedComponent;
});

describe('routes', () => {
  it('should return the correct routes', () => {
    const routes = require('./index').default;

    expect(routes).toMatchSnapshot();
  });
});
