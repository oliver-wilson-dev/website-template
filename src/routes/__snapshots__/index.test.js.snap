// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`routes should return the correct routes 1`] = `
Object {
  "about": Object {
    "component": [Function],
    "route": "/about",
  },
  "contact": Object {
    "component": [Function],
    "route": "/contact",
  },
  "home": Object {
    "component": [Function],
    "route": "/",
  },
}
`;
